releng/models.py

```diff
@@ -47,7 +47,7 @@ def iso_url(self):
         return "iso/%s/archlinux-%s-x86_64.iso" % (self.version, self.version)
 
     def tarball_url(self):
-        return "iso/%s/archlinux-bootstrap-%s-x86_64.tar.gz" % (self.version, self.version)
+        return "iso/%s/archlinux-bootstrap-%s-x86_64.tar.zst" % (self.version, self.version)
```

Comment:
Hm, wouldn't this also change for all currently available (and those not yet being zstd compressed)?

Reply:
`tarball_url` is used in only one place.
https://github.com/archlinux/archweb/blob/8679a0c0976a92c76fa307bba8ba0a1d2ea320df/templates/public/download.html#L117